src/marqo/tensor_search/tensor_search.py

```diff
@@ -1946,11 +1946,9 @@ def get_query_vectors_from_jobs(
                 try:
                     merged_vector = normalize_vector(merged_vector)
                 except core_exceptions.ZeroMagnitudeVectorError as e:
-                    tb = traceback.extract_tb(e.__traceback__)[-1]
                     raise core_exceptions.ZeroMagnitudeVectorError(
-                        f"Zero magnitude vector detected at file name: {tb.filename}, line number: {tb.lineno}, inside function name: {tb.name}. "
+                        f"Magnitude of combined query and context vectors cannot be zero. "
```

Comment:
Good error message, we can do a few things to increase clarity here:
1. Output the query and context vectors back to the user
2. Add a link to docs (we have an existing pattern for this, using the `marqo_docs.py` module)
3. Let's add this zero magnitude limitation to the docs

Reply:
1. Discussed this with @farshidz and I agree that this may be too verbose of an error message. Also we're not sure how it would provide value to our customers
2. Will be done in the next commit
3. Raised a PR for docs update: https://github.com/marqo-ai/marqodocs/pull/327